bigbone/src/main/kotlin/social/bigbone/api/entity/admin/AdminAccount.kt

```diff
@@ -0,0 +1,154 @@
+package social.bigbone.api.entity.admin
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import social.bigbone.DateTimeSerializer
+import social.bigbone.PrecisionDateTime
+import social.bigbone.api.entity.Account
+import social.bigbone.api.entity.Role
+
+/**
+ * Admin-level information about a given account.
+ */
+@Serializable
+data class AdminAccount(
+    /**
+     * The ID of the account in the database.
+     * String cast from an Integer, but not guaranteed to be a number.
+     */
+    @SerialName("id")
+    val id: String,
+
+    /**
+     * The username of the account.
+     */
+    @SerialName("username")
+    val username: String,
+
+    /**
+     * The domain of the account, if it is remote.
+     * Will be null for local accounts.
+     */
+    @SerialName("domain")
+    val domain: String? = null,
+
+    /**
+     * When the account was first discovered.
+     */
+    @SerialName("created_at")
+    @Serializable(with = DateTimeSerializer::class)
+    val createdAt: PrecisionDateTime = PrecisionDateTime.InvalidPrecisionDateTime.Unavailable,
+
+    /**
+     * The email address associated with the account.
+     */
+    @SerialName("email")
+    val email: String,
+
+    /**
+     * The IP address last used to log in to this account.
+     */
+    @SerialName("ip")
+    val ip: String? = null,
+
+    /**
+     * All known IP addresses associated with this account.
+     */
+    @SerialName("ips")
+    val adminIPs: List<IP>,
+
+    /**
+     *
+     */
+    @SerialName("role")
+    val role: Role,
+
+    /**
+     * Whether the account has confirmed their email address.
+     */
+    @SerialName("confirmed")
+    val confirmed: Boolean,
+
+    /**
+     * Whether the account is currently suspended.
+     */
+    @SerialName("suspended")
+    val suspended: Boolean,
+
+    /**
+     * Whether the account is currently silenced.
+     */
+    @SerialName("silenced")
+    val silenced: Boolean,
+
+    /**
+     * Whether the account is currently disabled.
+     */
+    @SerialName("disabled")
+    val disabled: Boolean,
+
+    /**
+     * Whether the account is currently approved.
+     */
+    @SerialName("approved")
+    val approved: Boolean,
+
+    /**
+     * Whether these account’s post are currently flagged sensitive.
+     */
+    @SerialName("sensitized")
+    val sensitized: Boolean? = null,
```

Comment:
Not mentioned in documentation but available in data returned by some methods, so I added it here, but as a nullable value with null default.